### DISCUSSION TITLE
Mount more persistent volumes

### DISCUSSION THREAD

**Discussion by d3bt3ch** [cloudnative-pg/cloudnative-pg#1800] [Ideas]
When can we expect the feature to `mount more persistent volumes` so that we can use `postgresql` `tablespaces`? This is a very important for large databases.

**Comment by wvengen** [comment]
I think this has been supported since Dec 2023 - https://cloudnative-pg.io/documentation/1.24/tablespaces/
See also e.g. https://github.com/cloudnative-pg/cloudnative-pg/pull/3410, https://github.com/cloudnative-pg/cloudnative-pg/pull/3464.
